streaming/text/convert/enwiki/mds/pick_eval_samples.py

```diff
@@ -7,17 +7,46 @@
 
 def parse_args():
     args = ArgumentParser()
-    args.add_argument('--input_dir', type=str, required=True)
-    args.add_argument('--output_dir', type=str, required=True)
-    args.add_argument('--compression', type=str, default='zstd:16')
-    args.add_argument('--hashes', type=str, default='sha1,xxh3_64')
-    args.add_argument('--size_limit', type=int, default=1 << 26)
-    args.add_argument('--num_examples_to_pick', type=int, default=10000)
+    args.add_argument(
+        '--in_root',
```

Comment:
you need to update the caller too

Reply:
Fixed in https://github.com/mosaicml/streaming/pull/171/commits/00c1e6d5f746940e20491297a395cc00e041ea37.